diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
index 81e1c49..b4df4f7 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
@@ -189,7 +189,7 @@ function schemaKeyCompare(k1, k2) {
     } else if (k2 in schemaKeySortOrder) {
         return 1;
     }
-    return k1.localeCompare(k2);
+    return 1; // Do not sort any other keys
 }
 
 /**
@@ -772,11 +772,13 @@ async function materializeSchemaToPath(schemaDirectory, schema, options = {}) {
                 // Just in case, force remove (ignore errors) latestSchemaPath
                 // before writing it.
                 await fse.rm(latestSchemaPath, { force: true });
+                /* See ULIMS-1091 for reason disabled
                 await writeObject(schema, latestSchemaPath, contentType);
                 log.info(
                     `Materialized latest schema at ${latestSchemaPath} with version ${version}.`
                 );
                 materializedFiles.push(latestSchemaPath);
+                */
             } else {
                 log.info(
                     '--dry-run: Would have materialized latest schema at ' +
diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
index 9853233..68066f5 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
@@ -60,7 +60,7 @@ function declareTests(options = { logLevel: 'warn' }) {
                 const schemaDir = path.dirname(schemaInfos[0].path);
 
                 it(`must contain ${options.currentName} schema file`, () => {
-                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')));
+                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')) | fs.existsSync(path.join(schemaDir, 'current.json')) );
                 });
 
                 // TODO: Is this a test we really need?
