diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
index acd2991..15da16e 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
@@ -212,7 +212,7 @@ function jsonStableStringifySchemaCompare(kv1, kv2) {
  */
 const serializers = {
     yaml: (obj) => { return yaml.dump(obj, { sortKeys: schemaKeyCompare }); },
-    json: (obj) => { return jsonStableStringify(obj, { cmp: jsonStableStringifySchemaCompare, space: 2 }); },
+    json: (obj) => { return jsonStableStringify(obj, { cmp: () => 1, space: 2 }); },
 };
 
 /**
diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
index b09a2c1..ea0d2c8 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
@@ -60,7 +60,7 @@ function declareTests(options = { logLevel: 'warn' }) {
                 const schemaDir = path.dirname(schemaInfos[0].path);
 
                 it(`must contain ${options.currentName} schema file`, () => {
-                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')));
+                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')) | fs.existsSync(path.join(schemaDir, 'current.json')) );
                 });
 
                 // TODO: Is this a test we really need?
