diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
index 81e1c49..4a4e894 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/jsonschema-tools.js
@@ -189,7 +189,7 @@ function schemaKeyCompare(k1, k2) {
     } else if (k2 in schemaKeySortOrder) {
         return 1;
     }
-    return k1.localeCompare(k2);
+    return 1; // Do not sort any other keys
 }
 
 /**
diff --git a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
index 9853233..68066f5 100644
--- a/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
+++ b/node_modules/@wikimedia/jsonschema-tools/lib/tests/structure.js
@@ -60,7 +60,7 @@ function declareTests(options = { logLevel: 'warn' }) {
                 const schemaDir = path.dirname(schemaInfos[0].path);
 
                 it(`must contain ${options.currentName} schema file`, () => {
-                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')));
+                    assert.ok(fs.existsSync(path.join(schemaDir, 'current.yaml')) | fs.existsSync(path.join(schemaDir, 'current.json')) );
                 });
 
                 // TODO: Is this a test we really need?
